pkg/k8sclient/clients.go

```diff
@@ -48,7 +48,7 @@ func NewClients(config ClientsConfig) (*Clients, error) {
 	}
 
 	if config.KubeConfigPath == "" && config.RestConfig == nil {
-		return nil, microerror.Maskf(invalidConfigError, "%T.KubeConfigPath or %T.RestConfig must not be empty", config, config)
+		return nil, microerror.Maskf(invalidConfigError, "%T.KubeConfigPath or %T.RestConfig must not be empty (KubeConfigPath is empty: %t; RestConfig is empty: %t)", config, config, config.KubeConfigPath == "", config.RestConfig == nil)
```

Comment:
The condition is `&&`. That means that both are empty. The error message might just be wrong.

Reply:
Oh snap, I missed that. Updated error messages :)

Reply:
When your fingers are faster than your brain :grimacing: